yaml/simulator.yaml

```diff
@@ -21,6 +21,7 @@
              -f <cwd>/files.f -full64
              -l <out>/compile.log
              -LDFLAGS '-Wl,--no-as-needed'
+             -CFLAGS '--std=c99 -fno-extended-identifiers'
```

Comment:
nit: perhaps good to add some comment for this option

Reply:
done.